modules/hydra-master-main.nix

```diff
@@ -93,37 +68,28 @@ in {
         input-output-hk = ${builtins.readFile ../static/github_token}
       </github_authorization>
       <githubstatus>
-        jobs = serokell:iohk-nixops.*
+        jobs = iohk:iohk-nixops.*
         inputs = jobsets
         excludeBuildFromContext = 1
       </githubstatus>
       <githubstatus>
-        jobs = serokell:cardano.*
+        jobs = iohk:cardano.*
         inputs = cardano
         excludeBuildFromContext = 1
       </githubstatus>
       <githubstatus>
-        jobs = serokell:daedalus.*:tests\..*
+        jobs = iohk:daedalus.*:tests\..*
```

Comment:
once this change is deployed, hydra will cease to report status's to all PR's until the project name is fixed in the hydra state (the webui config)

Reply:
This is one I was hesitant about as well. Do we end up losing history if we do this? Do we need some sort of db migration path to make this change without losing history?

Reply:
yeah, all history (including hydra metrics) would likely be lost, i dont believe hydra officially supports a rename, so you would have to create a new project, which starts fresh
declarative jobs would at least re-make every PR jobset, but all history will be missing